src/store/flow/block-types/Core_OutputBlockStore.ts

```diff
@@ -1,61 +1,39 @@
-import {ActionTree, GetterTree, MutationTree} from 'vuex'
+import {ActionTree, Module} from 'vuex'
 import {IRootState} from '@/store'
-import {IBlock, IBlockExit} from '@floip/flow-runner'
-import {IdGeneratorUuidV4} from '@floip/flow-runner/dist/domain/IdGeneratorUuidV4'
+import {IBlock} from '@floip/flow-runner'
 import {IOutputBlock} from '@floip/flow-runner/src/model/block/IOutputBlock'
-import {defaultsDeep} from 'lodash'
-import {validateCommunityBlock} from '@/store/validation/validationHelpers'
-import {IFlowsState} from '../index'
+import {cloneDeep} from 'lodash'
+import BaseStore, {actions as baseActions, IEmptyState} from '@/store/flow/block-types/BaseBlock'
 
 export const BLOCK_TYPE = 'Core.Output'
 
-export const getters: GetterTree<IFlowsState, IRootState> = {}
+const actions: ActionTree<IEmptyState, IRootState> = {
+  ...baseActions,
 
-export const mutations: MutationTree<IFlowsState> = {}
-
-export const actions: ActionTree<IFlowsState, IRootState> = {
   async editOutputExpression({commit}, {blockId, value}: { blockId: string, value: string }): Promise<string> {
     commit('flow/block_updateConfig', {blockId, newConfig: {value}}, {root: true})
     return value
   },
-  async createWith({dispatch}, {props}: { props: { uuid: string } & Partial<IOutputBlock> }) {
-    const exits: IBlockExit[] = [
-      await dispatch('flow/block_createBlockDefaultExitWith', {
-        props: ({
-          uuid: await (new IdGeneratorUuidV4()).generate(),
-        }) as IBlockExit,
-      }, {root: true}),
-    ]
 
-    return defaultsDeep(props, {
-      type: BLOCK_TYPE,
-      name: '',
-      label: '',
-      semantic_label: '',
-      config: {
-        value: null,
-      },
-      exits,
-      tags: [],
-      vendor_metadata: {},
-    })
+  async createWith({dispatch}, {props}: { props: { uuid: string } & Partial<IOutputBlock> }) {
+    props.type = BLOCK_TYPE
+    props.config = {
+      value: '@(null)',
```

Comment:
@bulatgab , I think we should keep `null` like in master https://github.com/FLOIP/flow-builder/blob/master/src/store/flow/block-types/Core_OutputBlockStore.ts#L36, that will provide us the right validation message at the beginning. Your thoughts ?
<img width="354" alt="Screen Shot 2022-05-25 at 9 07 44 AM" src="https://user-images.githubusercontent.com/68745918/170295577-ed0bb289-f06d-4080-9142-d7cb5a6f2f5f.png">

Reply:
Upd: changed to `value: null` as suggested